src/ngraph_mark_for_clustering.cc

```diff
@@ -381,6 +381,11 @@ Status MarkForClustering(Graph* graph) {
         NGRAPH_VLOG(4) << "Rejecting: " << node->name() << "["
                        << node->type_string() << "]";
       }
+
+      if (config::is_logging_placement()) {
+        const char* status = confirmed ? "Accepting: " : "Rejecting: ";
```

Comment:
If we want to log the final cluster assignment, it might make more sense to do this in the `DeassignClusters` pass.